snuba/utils/streams/topics.py

```diff
@@ -41,13 +41,15 @@ class Topic(Enum):
     GENERIC_METRICS_DISTRIBUTIONS_COMMIT_LOG = (
         "snuba-generic-metrics-distributions-commit-log"
     )
+    SEARCH_ISSUES = "eventstream-issue-platform"
```

Comment:
Should this just be `issue-platform`? Afaik, all events go through the eventstream so I'm not sure that part is particularly meaningful here.

Reply:
@wedamija what do you think about `ISSUE_PLATFORM = "issue-platform"` or `ISSUE_PLATFORM = "issue-platform-occurrence"`?

Reply:
let's call it `issue-occurrences`? Or maybe `generic-events` to match what we're calling the events.

Reply:
Ok. To be consistent, I'll go with `generic-events`.